source/robot_model/src/Model.cpp

```diff
@@ -150,6 +150,49 @@ state_representation::Jacobian Model::compute_jacobian(const state_representatio
   return this->compute_jacobian(joint_positions, frame_id);
 }
 
+state_representation::Jacobian Model::compute_jacobian_time_derivative(const state_representation::JointPositions& joint_positions,
```

Comment:
This all look nice but I will have to disagree on the return type. This is not actually a `Jacobian` per se and there are not JacobianTimeDerivative yet. I would change for a `Eigen::MatrixXd` for now.